components/builder-originsrv/src/migrations/origin_packages.rs

```diff
@@ -267,5 +317,32 @@ pub fn migrate(migrator: &mut Migrator) -> Result<()> {
                         RETURN;
                     END
                     $$ LANGUAGE plpgsql STABLE"#)?;
+    migrator.migrate("originsrv",
+                     r#"CREATE OR REPLACE FUNCTION get_origin_packages_for_version_v1 (
+                            op_ident text,
```

Comment:
I don't disagree, but it looks like `op_ident` is what's used pretty much everywhere, including where the identifier is partial. I'd rather keep to the existing pattern for this PR and submit a separate one that addresses the naming of these params and related cleanup more specifically.